conf/nginx/templates/nginx.conf.web.clamav.signature.provider.template

```diff
@@ -0,0 +1,23 @@
+# Carbonio ClamAV Signature Provider proxy configuration
+#
+server
+{
+    listen 127.78.0.21:10000;
+    access_log /opt/zextras/log/nginx_clamav_signature_provider_access.log;
+
+
+    location / {
+        add_header Content-Type text/plain;
+        return 200 'Ok';
+        allow 127.0.0.1;
+        allow ::1;
+        deny all;
+    }
+
+    location ^~ /av_signature {
+        alias /opt/zextras/av_signature/;
+        allow 127.0.0.1;
+        allow ::1;
```

Comment:
Careful allowing only localhost as the request from envoy may come from other IPs, unless we are 100% sure

Reply:
I don't know, but in the tests i performed I saw all reuqests came from localhost. 
I tested on multiserver setup where proxy and clamav node are separate. Sending requests to localbind address of clamav amde the request to the proxy come from localhost. 
Let me know if there are ways to simulate situations where IP could be different.

Reply:
I don't know about any specific situations, it's just that the sidecar IP is 0.0.0.0, I never monitored where the request comes from though. I'd say it's ok but let's keep an eye open :eyes:

Reply:
sure thing :+1: